realm/realm-library/src/main/java/io/realm/MutableRealmInteger.java

```diff
@@ -22,72 +22,69 @@
 
 
 /**
- * A MutableRealmInteger is a mutable, {@link java.lang.Long}-like numeric quantity.
- * It behaves almost exactly as a reference to a {@link java.lang.Long}. More specifically:
+ * A {@code MutableRealmInteger} is a mutable, {@link Long}-like numeric quantity.
+ * It behaves almost exactly as a reference to a {@link Long}. More specifically:
  * <ul>
- * <li>A MutableRealmInteger may have the value {@code null}.</li>
- * <li>The {@code .equals} operator compares the contained Long values. null-valued MutableRealmInteger are {@code .equals}</li>
- * <li>The {@code .compareTo} operator compares the contained Long values.  It considers {@code null} &lt; any value.</li>
- * <li>The {@code .increment} and {@code .decrement} operators throw {@code IllegalStateException} when applied to a null-valued MutableRealmInteger.</li>
+ * <li>A {@code MutableRealmInteger} may have the value {@code null}.</li>
+ * <li>The {@link #equals} operator compares the contained {@link Long} values. {@code null}-valued {@code MutableRealmInteger} are {@code .equals}</li>
```

Comment:
I felt this was clearer.  "are equal" could mean "==" or ".equals".  I mean, specifically, the latter, so I said that.